.github/workflows/presubmit.yml

```diff
@@ -7,6 +7,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - run: |  # Needed for presubmit to work.
+        git fetch origin master --depth 1
```

Comment:
dont understand this, will let @oliverchang to review.
why not git pull ?

Reply:
I'm assuming this is for speed?

Reply:
The `--depth 1` part is but given the weird way GitHub actions checksout a repo `git pull` won't work.
You can see for yourself [here](https://github.com/google/fuzzbench/pull/97/checks?check_run_id=501465871), but GitHub actions does the following to checkout a pull request:
```
git init
/usr/bin/git remote add origin https://github.com/google/fuzzbench
/usr/bin/git config --local gc.auto 0
/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +0c98f7fd8841b8e5c5cb01a8582f0e6a17890e74:refs/remotes/pull/97/merge
/usr/bin/git checkout --progress --force refs/remotes/pull/97/merge
```

As a result, some things that work when a repo is cloned using `git clone` don't hold so I'm working around it.
But there's a fair amount here I don't fully understand.